.vscode/launch.json

```diff
@@ -29,9 +29,13 @@
         "-pipeID", "SendReceive${env:HOME}/fwrepo/fw/DistFiles/Projectsobtain"
       ],
       "env": {
+        // Send and receive to the QA LanguageForge server instead of production, if desired.
+        "LANGUAGEFORGESERVER": "-qa.languageforge.org",
```

Comment:
Is the dash in front of qa intentional?

Reply:
Thank you. Yes. I think it may be combined with another string. It was copied this way from FieldWorks.